riscoss-platform-risk-analyser/api/src/main/java/eu/riscoss/reasoner/ReasoningLibrary.java

```diff
@@ -9,58 +9,26 @@
 {
 	private static final ReasoningLibrary instance = new ReasoningLibrary();
 	
-	public static ReasoningLibrary get()
-	{
+	public static ReasoningLibrary get() {
 		return instance;
 	}
 	
-	Map<String,RiskAnalysisStructure> engines = new HashMap<String,RiskAnalysisStructure>();
+	Map<String,RiskAnalysisEngine> engines = new HashMap<>();
 	
 	List<String> classes = new ArrayList<String>();
 	
 	private ReasoningLibrary() {
-		for( String className : new ClassList( "eu.riscoss.reasoner.impl" ) ) {
-			try {
-				Class<?> cls = Class.forName( className );
-				if( RiskAnalysisEngine.class.isAssignableFrom( cls ) ) {
-					classes.add( className );
-				}
-			}
-			catch( Exception ex ) {}
-		}
+		classes.add( "eu.riscoss.reasoner.CompoundAnalysisEngine" );
+		classes.add( "eu.riscoss.reasoner.FBKRiskAnalysisEngine" );
 	}
 	
 	public RiskAnalysisEngine createRiskAnalysisEngine() {
-		try {
-			return (RiskAnalysisEngine) Class.forName( "eu.riscoss.reasoner.CompoundAnalysisEngine" ).newInstance();
-		}
-		catch( Exception ex ) {}
-		
-		return null;
-	}
-	
-	// TODO: Remove?
-	public RiskAnalysisStructure createRiskStructure() {
-		try {
-			return (RiskAnalysisStructure) Class.forName( classes.get( 0 ) ).newInstance();
+		for( String clsName : classes ) {
+			try {
+				return (RiskAnalysisEngine) Class.forName( clsName ).newInstance();
+			}
+			catch( Exception ex ) {}
```

Comment:
If you throw a RuntimeException here or change this to a checked exception, it will come out in the logs in a nice way.